frontend/src/views/Schools.svelte

```diff
@@ -46,11 +47,26 @@
       <ul class="list-group w-100">
         {#each schools as school}
           <li class="list-group-item d-flex justify-content-between align-items-center">
+            {#if school.isServiceEnabled}
+            <a 
+              href="/"
+              onclick={() => setCurrentSchool(school)}
+            >
             <div>
-              {school.displayName}, sist oppdatert {new Date(school.updatedAt).toLocaleString(
-                'no-NO'
-              )}
+              <span class="school-name">
+                {school.displayName}, sist oppdatert {new Date(school.updatedAt).toLocaleString('no-NO')}
+              </span>
             </div>
+          </a>
+            {:else}
+            <div class="text-start d-flex align-items-center">
+              <div>
+                <span class="inactive-school">
+                  {school.displayName}, sist oppdatert {new Date(school.updatedAt).toLocaleString('no-NO')}
+                </span>
+              </div>
+            </div>
```

Comment:
There are more HTML elements here than needed, I think? Can you make it work with a single div?